skill-tree /src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -0,0 +1,229 @@
+    public void testAPIReturns400_OnInvalidStatusPassed(){
+        Response response = given()
+                .queryParam("status","APPROVAL")
+                .get("/endorsements");
+
+        response.then()
+                .statusCode(400)
+                .body("code", equalTo(400))
+                .body("message", equalTo("Invalid status passed"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 200 on endorsement creation")
+    public void testAPIReturns200_OnEndorsementCreation() throws JsonProcessingException {
+
+        String requestBody = objectMapper.writeValueAsString(EndorsementCreationRequest.builder()
+                .endorserUserId("u-1")
+                .endorseeUserId("u-2")
+                .skillName("Java")
+                .description("This is endorsed for the skill showcased in Skill tree")
+                .endorsementBehaviour(Behaviour.POSITIVE)
+                .build());
+
+        Response response = given()
+                .contentType("application/json")
+                .body(requestBody)
+                .post("/endorsements");
+        response.then()
+                .statusCode(200)
+                .body("code", equalTo(200))
+                .body("message", equalTo("Created endorsement sucessfully"));
+
+
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 200 on endorsement updation")
+    public void testAPIReturns200_OnEndorsementsUpdation(){
+        String endorsementId = "e-1";
+        Response response = given()
+                .pathParam("endorsementId", endorsementId)
+                .patch("/endorsements");
+
+        response.then()
+                .statusCode(200)
+                .body("code",equalTo(200))
+                .body("message", equalTo("Updated endorsement sucessfully"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 400 on invalid endorsementId passed")
+    public void testAPIReturns400_OnInvalidParameterPassed(){
+        String endorsementId = "randomId";
+        Response response = given()
+                .pathParam("endorsementId", endorsementId)
+                .patch("/endorsements");
+
+        response.then()
+                .statusCode(400)
+                .body("code",equalTo(400))
+                .body("message", equalTo("Invalid endorsementId passed"));
+    }
+
+    @Test
+    @Disabled
+    @DisplayName("Return 200 on endorsements updation")
+    public void testAPIReturns200_OnEndorsementGivenId(){
+        String endorsementId = "e-1";
+        Response response = given()
+                .pathParam("endorsementId", endorsementId)
+                .patch("/endorsements");
+
+        /*
+        {
+  "endorseeId": "user-1",
+  "skillName": "Java",
+  "status": "APPROVED",
+  "endorsementType": "POSITIVE",
+  "endorsersList": [
+    {
+      "endorserId": "user-2",
+      "description": "string",
+      "userType": "NORMAL_USER"
+    }
+  ]
+}
+         */
```

Comment:
Is it for reference purpose ? or by mistake missed to remove. if its for reference can we format it please.

Reply:
Its for reference, will get it formatted